dom.bs

```diff
@@ -33,7 +33,9 @@ urlPrefix: https://tc39.github.io/ecma262/#; spec: ECMASCRIPT
  text: Realm; url: realm; type: dfn
 urlPrefix: https://w3c.github.io/hr-time/#; spec: HR-TIME
  type:typedef; urlPrefix: dom-; text: DOMHighResTimeStamp
- type:dfn; text: time origin
+ type:dfn;
+ text: time origin;
+ url: dfn-time-origin
```

Comment:
These three have to be on the same line.

Reply:
Fair to ask if that is for style?

Just to answer for why I did that, I thought about how that is an exact (or should I say verbatim?) copy of the formatting standard for former/earlier lines, in fact the original <pre> section is the readable column with linebreak style, not the inlined/minified/abbreviated style, from the very beginning of the document code in question.

Reply:
https://tabatkins.github.io/bikeshed/#infotree

Reply:
Thank you for the link.

So why are "`type`, `text`, `url`" otherwise herein given separate/individual lines?

I hear the documentation says otherwise as well (showing single and then nesting format), so is our spec in conflict, or is their specification tooling causing the problem, er,... still fair to ask if that's for style yo?

Reply:
They are only on separate lines as well as appropriately indented when they are shared, as per the format. If you mean something else then please give an example, as it isn't entirely clear from your message where the confusion stems from.

Reply:
Hmm. Fair. I really think about how some coders might allow that for style.

I deferred to you however and made the correction to our specification, I did not mean my <style> to be mean, questioning about making a style statement.